src/Components/Server/src/Circuits/CircuitHost.cs

```diff
@@ -104,21 +104,32 @@ public Task<IEnumerable<string>> PrerenderComponentAsync(Type componentType, Par
             });
         }
 
-        public async Task InitializeAsync(CancellationToken cancellationToken)
```

Comment:
It would be really nice to leave the signature alone and have the hub be the one to fire-and-forget this task. When we have async code we need to be able to await it in a test or errors will creep in. 

Basically have a big, thick, gooey async method that returns a task, and have a small stub (`StartCircuit` in this case) do `_ = InitializeAsync()`

Reply:
Sure, done.